client/http-client.js

```diff
@@ -26,6 +27,14 @@ module.exports = (clientConfig) => {
     proxy: undefined
   }
 
+  const timeout = clientConfig.requestTimeout || 10000
+  const axiosClient = axios.create({
```

Comment:
why do we use axios here? It doesn't make sense to use axios and node-fetch.

Reply:
Yes, we want to switch to axios, also required if we want to add the import API to the SDK. I just couldn't do it all in this PR :)